moped-editor/src/utils/activityLogFormatters/mopedComponentsActivity.js

```diff
@@ -21,9 +21,12 @@ export const formatComponentsActivity = (
   const changeIcon = <RoomOutlinedIcon />;
   const component =
     componentList[change.record_data.event.data.new.component_id];
+  const componentID = change.record_data.event.data.new.project_component_id;
+  const componentLink = `/moped/projects/${projectId}?tab=map&project_component_id=${componentID}`;
   const componentText = {
-    text: component,
+    text: `${component} (#${componentID})`,
     style: "boldText",
+    link: componentLink
```

Comment:
I think it is great the way it is, but I wondered if we should link when a component is deleted since the map will always show a "could not be found" alert? I can convince myself either way. 🤷

<img width="348" alt="Screenshot 2024-01-12 at 10 10 34 AM" src="https://github.com/cityofaustin/atd-moped/assets/37249039/c7513f2c-80ce-47a4-bb48-84763241b27c">